pytoda/smiles/transforms.py

```diff
@@ -357,7 +501,7 @@ def __call__tensor(self, smiles_numerical: torch.Tensor) -> str:
         Returns:
             str: randomized SMILES representation.
         """
-
+        # TODO expects 2D Tensor as input, check for 1D.
```

Comment:
this check should be done

Reply:
the check was actually done in the method calling this 👍